package.json

```diff
@@ -32,32 +32,24 @@
   "homepage": "https://github.com/silverstripe/silverstripe-campaign-admin",
   "dependencies": {
     "bootstrap": "4.0.0-beta",
+    "classnames": "^2.2.5",
     "deep-freeze-strict": "^1.1.1",
     "react": "^15.3.1",
+    "react-addons-test-utils": "^15.3.1",
     "react-bootstrap-ss": "^0.30.9",
     "react-dom": "^15.3.1",
     "react-redux": "^4.4.1",
     "react-router": "^2.4.1",
     "react-router-redux": "^4.0.5",
     "redux": "https://registry.npmjs.org/redux/-/redux-3.0.5.tgz",
-    "redux-form": "^6.0.2"
+    "redux-form": "^6.0.2",
+    "redux-mock-store": "^1.2.3",
+    "redux-thunk": "^2.2.0"
   },
   "devDependencies": {
-    "@silverstripe/webpack-config": "^0.2.7",
-    "babel-core": "^6.7.4",
+    "@silverstripe/webpack-config": "file:../../webpack-config",
```

Comment:
For development purposes, to replace with the proper `0.3.0` version when that is published on the `webpack-config` package

Reply:
Can you change it back to "^0.3.0" since changes on `webpack-config` has been merged?